### ISSUE TITLE
operator_usage_whitespace detects invalid violation with nested generics

### ISSUE THREAD

**Issue by NachoSoto** [realm/SwiftLint#1341]
```swift
return SignalProducer<Signal<Value, Error>, Error>([ self.signal, next ]).flatten(.concat)
```

This is pointing to the second `>`, which is obviously not the `>` operator :P

**Comment by marcelofabri** [comment]
I was able to reproduce it with 0.16.1.
